downstream/modules/platform/proc-controller-configure-secret-lookups.adoc

```diff
@@ -13,15 +13,15 @@ Use the following procedure to use {ControllerName} to configure and use each of
 .Procedure
 . Create an external credential for authenticating with the secret management system. At minimum, give a name for the external credential and select one of the following for the *Credential Type* field:
 +
-* xref:ref-aws-secrets-manager-lookup[AWS Secrets Manager Lookup]
-* xref:ref-centrify-vault-lookup[Centrify Vault Credential Provider Lookup]
-* xref:ref-cyberark-ccp-lookup[CyberArk Central Credential Provider (CCP) Lookup]
-* xref:ref-cyberark-conjur-lookup[CyberArk Conjur Secrets Manager Lookup]
-* xref:ref-hashicorp-vault-lookup[HashiCorp Vault Secret Lookup]
-* xref:ref-hashicorp-signed-ssh[HashiCorp Vault Signed SSH]
-* xref:ref-azure-key-vault-lookup[{Azure} Key Vault]
-* xref:ref-thycotic-devops-vault[Thycotic DevOps Secrets Vault]
-* xref:ref-thycotic-secret-server[Thycotic Secret Server]
+* TBD[AWS Secrets Manager Lookup]
```

Comment:
I'm not positive but wouldn't these links remain the same since we're going to include this info in the controller docs? Or, have you've moved them to a different document and are waiting to figure out the new URLs for links? I think Felicia finished putting the docs with title changes into Pantheon so you might be able to resolve these now. Let me know if you need help.

Reply:
I moved Secrets to the Admin Guide. I did work out what some of the links would be. I didn't get round to adding them. I wanted to check the Credentials details first.